applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -67,13 +73,36 @@ def _create_solution_scheme(self):
         elif(scheme_type == "bossak"):
             damp_factor_m = self.dynamic_settings["damp_factor_m"].GetDouble()
             mechanical_scheme = KratosMultiphysics.ResidualBasedBossakDisplacementScheme(damp_factor_m)
+        elif("bdf" in scheme_type or scheme_type == "backward_euler"):
+            if (scheme_type == "backward_euler"):
+                order = 1
+            else:
+                order = int(scheme_type.replace("bdf",""))
```

Comment:
I don't understand what you are doing here

Reply:
Same as above